samples/SocketsSample/EndPoints/MessagesEndPoint.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Sockets;
+
+namespace SocketsSample.EndPoints
+{
+    public class MessagesEndPoint : MessagingEndPoint
+    {
+        public ConnectionList Connections { get; } = new ConnectionList();
```

Comment:
Yeah, I don't disagree. I was wondering if there was ever a purpose for a heterogeneous collection of connections, but I don't know that there is.